lms/djangoapps/instructor/views/api.py

```diff
@@ -946,6 +947,16 @@ def modify_access(request, course_id):
         'action': action,
         'success': 'yes',
     }
+    # manage student and admin transitions for license enforcement.
+    if run_extension_point('PCO_IS_LICENSED_CCX',course_id=course_id):
```

Comment:
Let's check if course licensing is enable before calling anything. @anfbermudezme 
```suggestion
    if run_extension_point('PCO_ENABLE_COURSE_LICENSING'):
```